foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -8,7 +8,7 @@ Project Griffon is a beta product. To use it, you must accept the terms on [http
 
 To get started with [Project Griffon](../../beta/project-griffon/) in your app, you'll need to:
 
-1. Install the AEP Assurance extension in Experience Platform Launch
+1. Install the AEP Assurance extension in [Experience Platform Launch](https://launch.adobe.com/)
```

Comment:
Not sure we are still calling it `Experience Platform Launch`. The page title says `Adobe Experience Platform Data Collection`.

Reply:
Fixed to keep consistent with page title.